zipkin-web/src/main/resources/app/js/page/default.js

```diff
@@ -52,12 +52,15 @@ define(
           const endTs = query.endTs || new Date().getTime();
           const serviceName = query.serviceName || '';
           const annotationQuery = query.annotationQuery || '';
+          const queryWasPerformed = serviceName && serviceName.length > 0;
```

Comment:
perhaps is would be better to define a flag based on `modelView.traces.length`?  What happens if I GET the home page with `?serviceName=x`?